custom-targets/git-ops/README.md

```diff
@@ -0,0 +1,81 @@
+| customTarget/gitPullRequestBody | No | The body of the pull request, if not provided then defaults to "Project: {project-num} Location: {location} Delivery Pipeline: {pipeline-id} Target: {target-id} Release: {release-id} Rollout: {rollout-id}" |
+| customTarget/gitEnableArgoSyncPoll | No | Whether to poll the sync status of the Argo Application. If `true` then the pull request is merged and the deployer polls the Argo Application until the the merged changes are synced. When enabled the following deploy parameters become required: `gitGKECluster`, `gitArgoApplication`, and `gitArgoNamespace` |
+| customTarget/gitGKECluster | No | The name of the GKE cluster hosting the Argo Application resource, required when `gitEnableArgoSyncPoll` is `true` |
+| customTarget/gitArgoApplication | No | The name of the Argo Application resource associated with the Git repository, required when `gitEnableArgoSyncPoll` is `true` |
+| customTarget/gitArgoNamespace | No | The namespace the Argo Application resource resides in, required when `gitEnableArgoSyncPoll` is `true` |
+| customTarget/gitArgoSyncTimeout | No | Duration to poll the sync status of the Argo Application, if not provided then defaults to 30 minutes |
+
+## Secret - Personal Access Token
+When using Github, a personal access token must be configured and uploaded to Secret Manager. When using Gitlab, a project access token can be configured and uploaded. The service account used in the target execution environment must be configured with the role `roles/secretmanager.secretAccessor` to read the token secret from Secret Manager.
+
+The Github PAT must be configured to have `Read and Write` access for `Contents` and `Pull Requests`. 
+
+The Gitlab PAT must be configured to use the role `Maintainer` with the `api` and `write_repository` permissions.
+
+
+<a name="build"></a>
+# Build the sample image and register a Custom Target Type for Terraform
+The `build_and_register.sh` script within this `git-ops` directory can be used to build the Git deployer image and register a Cloud Deploy custom target type that references the image. To use the script run the following command:
+
+```shell
+./build_and_register.sh -p $PROJECT_ID -r $REGION
+```
+
+The script does the following on your behalf:
+1. Create an Artifact Registry Repository
+2. Give the default compute service account access to the Repository
+3. Build the image and push it to the Repository
+4. Create a Cloud Storage bucket and within the bucket a skaffold configuration that references the image built
+5. Apply a custom target type for Git to Cloud Deploy that references the skaffold configuration in Cloud Storage
+
+# How the sample image works
+The Git deployer sample image is built to only handle deploy requests from Cloud Deploy. The expectation is that the default rendering process is performed by Cloud Deploy.
```

Comment:
"The Git deployer sample image is built to only handle deploy requests from Cloud Deploy."

I was confused by this. I thought it meant "it will not handle deploy requests from anyone besides Cloud Deploy".

How about:

"The Git deployer sample uses the default Cloud Deploy rendering process. After rendering, deploy operations are handled by this Git deployer."